src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +13,27 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    public static final String START_OF_WORD = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] result = new String[0];
+
+        try {
+            String text = Files.readString(Paths.get(fileName));
+            String[] words = text.toLowerCase().split(" ");
+            StringBuilder wordsFromW = new StringBuilder();
+            for (String i: words) {
+                if (i.startsWith(START_OF_WORD)) {
+                    wordsFromW.append(i.replaceAll("\\W", "")).append(" ");
+                }
+            }
+            if (wordsFromW.length() > 0) {
+                result = wordsFromW.toString().split(" ");
+            }
+            Arrays.sort(result);
+        } catch (IOException e) {
+            System.out.println("No files");
```

Comment:
here you basically do nothing with this exception

Reply:
done